tfhe/src/high_level_api/compressed_ciphertext_list.rs

```diff
@@ -76,7 +76,7 @@ impl CompressedCiphertextListBuilder {
 pub struct CompressedCiphertextList(crate::integer::ciphertext::CompressedCiphertextList);
 
 impl Named for CompressedCiphertextList {
-    const NAME: &'static str = "high_level_api::CompactCiphertextList";
+    const NAME: &'static str = "high_level_api::CompressedCiphertextList";
```

Comment:
should be backported in 0.7 when you get some time

Reply:
done in the release PR